src/main/java/org/concordion/api/AbstractCommandDecorator.java

```diff
@@ -11,26 +11,26 @@ public AbstractCommandDecorator(Command command) {
         this.command = command;
     }
 
-    public void setUp(final CommandCall commandCall, final Evaluator evaluator, final ResultRecorder resultRecorder) {
+    public void setUp(final CommandCall commandCall, final Evaluator evaluator, final ResultRecorder resultRecorder, final Fixture fixture) {
         process(commandCall, evaluator, resultRecorder, new Runnable() {
             public void run() {

```

Comment:
Have we moved to the version of Java that has Lambdas yet? If so, the whole 'new Runnable' thing isn't needed anymore.

Reply:
Not yet, still supporting JDK 7 currently (and checking with openjdk7 on Travis). Moving up can't be far away though.